src/profile-logic/import/dhat.js

```diff
@@ -253,11 +254,18 @@ export function attemptToConvertDhat(json: mixed): Profile | null {
   const bytesAtGmax: Bytes[] = [];
   const endBytes: Bytes[] = [];
 
+  // Maps prefixes to their descendents (more specific prefixes).
+  const postfix: Map<IndexIntoStackTable | null, IndexIntoStackTable[]> =
+    new Map();
+
   for (const pp of dhat.pps) {
     // Never reset the stackIndex, stack indexes always growing larger.
-    let stackIndex = -1;
+    let stackIndex = 0;
```

Comment:
I think this value isn't used outside of the loop now, so it can be moved below. Also look at the other comment about the same `stackIndex` variable below.

Reply:
This variable is still used one time after the loop, to tell the ending `stackIndex` of the PP entry to the `allocationsTable`!